extensions/wikia/WallNotifications/WallNotificationsExternalController.class.php

```diff
@@ -132,4 +132,33 @@ private function getUpdateWikiInternal( WallNotifications $wn, $wikiId, $isCross
 		$this->response->setCachePolicy( WikiaResponse::CACHE_PRIVATE );
 		$this->response->setCacheValidity( WikiaResponse::CACHE_DISABLED );
 	}
+
+	/**
+	 * Requested by Wall Notifications for cross-wiki notifications
+	 * @see WallNotificationEntity::loadDataFromRevIdOnWiki()
+	 */
+	public function getEntityData() {
+		$this->response->setCacheValidity( WikiaResponse::CACHE_DISABLED );
```

Comment:
This endpoint is only requested internally by ApiService and wasn't cached before